### PR TITLE
Change building env and thirdparty library for user to build as library

### PR DESCRIPTION
**Pull request by zsluedem**
Trying to fix this one https://github.com/Vid201/silius/issues/175

### CODE REVIEW

**Comment by zsluedem** [comment]
@Vid201 Temporary fix now. Ultimately would use alloy.

File: Makefile [hunk 1]
```diff
@@ -23,8 +23,8 @@ fetch-thirdparty:
 	git submodule update --init
 
 setup-thirdparty:
-	cd thirdparty/account-abstraction && yarn install --frozen-lockfile --immutable && yarn compile && cd ../..
-	cd thirdparty/bundler && yarn install --frozen-lockfile --immutable && yarn preprocess && cd ../..
+	cd crates/contracts/account-abstraction && yarn install --frozen-lockfile --immutable && yarn compile && cd ../..
```
Comment (Vid201) [inline Makefile:26]: Shouldn't be `cd ../../..` at the end?

**Review by Vid201 — CHANGES_REQUESTED** [review]
LGTM, this should solve the problem in the meantime

I am wondering if it's better to keep thirdparty folder -> meaning tests/thirdparty/bundler and crates/contracts/thirdparty/account-abstraction to indicate third party repos. Not sure if needed though

File: Makefile (hunk 1 above)
Comment (zsluedem) [reply Makefile:26]: yes. You are right

**Review by Vid201 — APPROVED** [review]
GJ

**Pull request merged**